app/lib/local_restrictions/local-restrictions.yaml

```diff
@@ -0,0 +1,442 @@
+---
+E06000003:
+  alert_level: 2
```

Comment:
Redcar and Cleveland are level 1 so we need to take this chunk out!

Reply:
Good spot, Sorted!!